nuheat/api.py

```diff
@@ -38,7 +40,11 @@ def authenticate(self):
             "Password": self.password,
             "application": "0"
         }
-        data = self.request(config.AUTH_URL, method="POST", data=post_data)
+        data = self.request(
+            url=util.get_auth_url(brand=self._brand),
```

Comment:
if taking my suggestion below, this could just be 

```suggestion
            url=self._auth_url,
```